src/net/l3/rarp.c

```diff
@@ -147,32 +141,30 @@ static int rarp_hnd_request(const struct arphdr *rarph,
 
 	/* declone sk_buff */
 	if (NULL == skb_declone(skb)) {
-		DBG(printk("rarp_hnd_request: no memory\n"));
+		log_error("rarp_hnd_request: no memory\n");
 		skb_free(skb);
 		return -ENOMEM;
 	}
 
-	DBG({
-		printk("rarp_hnd_request: send reply with ");
-		if (rarph->ar_pro == ntohs(ETH_P_IP)) {
-			struct in_addr in;
-			assert(rarph->ar_pln == sizeof in);
-			memcpy(&in, dst_paddr, sizeof in);
-			printk("%s", inet_ntoa(in));
-		}
-		else {
-			printk("unknown(%x)", htons(rarph->ar_pro));
-		}
-		if (rarph->ar_hrd == ntohs(ARP_HRD_ETHERNET)) {
-			assert(rarph->ar_hln == ETH_ALEN);
-			printk("[" MACADDR_FMT "]",
-				MACADDR_FMT_ARG(dst_haddr));
-		}
-		else {
-			printk("[unknown(%x)]", htons(rarph->ar_hrd));
-		}
-		printk("\n");
-	});
+	log_info("rarp_hnd_request: send reply with ");
```

Comment:
Is such change appropriate?

Anyway, `log_info` is obviously too severe for debug output.